src/main/java/frc/robot/utils/debugging/LoggedTunableNumber.java

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2024 FRC 6328
```

Comment:
When we pull a class in from another repo, we should include a link back to the original file.  This will allow us to find the file in case there are bug fixes or we need to track down better documentation.